pkg/eventprocessor/impl/executor/impl/txnscope_runsql_customfuncs_test.go

```diff
@@ -78,13 +78,6 @@ func TestCustomFunctionsWriteQuery(t *testing.T) {
 			newExecutorWithTable: newExecutorWithIntegerTable,
 			assertExpectation:    checkBlockNumberEq1,
 		},
-		{
```

Comment:
I deleted this test because it fails at different levels than other tests, and the `mustFail` does not give context to the assertion logic. This case is already tested in the parser, so I think it's fine.